.github/ISSUE_TEMPLATE/SlicingCrash.yaml

```diff
@@ -5,38 +5,24 @@ body:
 - type: markdown
   attributes:
     value: |      
-       ### ✨Try our improved Cura 5.7✨
-       Before filling out the report below, we want you to try the latest Cura 5.7. 
-       This version of Cura has become significantly more reliable and has an updated slicing engine that will automatically send a report to the Cura Team for analysis. 
-       #### [You can find the downloads here](https://github.com/Ultimaker/Cura/releases/latest) #### 
-       If you still encounter a crash you are still welcome to report the issue so we can use your model as a test case, you can find instructions on how to do that below.      
+       ### ✨Are you stuck? Have you tried these two things? ✨
+       1- Are you on a Cura version lower than Cura 5.7? We really recommend updating because it resolves a lot of slicing crashes!
+       2- Have you tried fixing the model with software that repairs 3d files and makes them watertight? 
```

Comment:
(Or downloaded a plugin from the Marketplace like https://marketplace.ultimaker.com/app/cura/plugins/fieldofview/MeshTools ;-) )

Reply:
Added 😊